dataflux_core/fast_list.py

```diff
@@ -163,8 +163,10 @@ def run(self) -> None:
             try:
                 list_blob_args = {
                     "max_results": self.max_results,
-                    "start_offset": self.start_range,
-                    "end_offset": self.end_range,
+                    "start_offset": self.prefix + self.start_range,
```

Comment:
If I remember correctly, when we initialize the workers other than the first one, we set the start and end offset to "None" and this tells them that they have no work to do. Does this impact that logic at all?